components/compositing/touch.rs

```diff
@@ -44,11 +46,6 @@ impl TouchPoint {
 pub enum TouchState {
     /// Not tracking any touch point
     Nothing,
-    /// A touchstart event was dispatched to the page, but the response wasn't received yet.
-    /// Contains the initial touch point.
-    WaitingForScript,
-    /// Script is consuming the current touch sequence; don't perform default actions.
-    DefaultPrevented,
```

Comment:
I think we might want to keep this state, we could perhaps transition to it after the `preventDefault()` from a `move` event arrives.